easybuild/easyconfigs/o/Oracle/Oracle-12.1.0.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = "PackedBinary"
+
+name = "Oracle"
+version = "12.1.0.2.0"
+
+homepage = 'https://www.oracle.com/technetwork/database/database-technologies/instant-client/overview/index.html'
+description = """Free, light-weight, and easily installed Oracle Database tools,
+                 libraries and SDKs for building and connecting applications to an Oracle Database instance."""
+
+toolchain = SYSTEM
+
+sources = [
```

Comment:
if sources can't be downloaded automatically, should provide `download_instructions`